aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -346,9 +357,10 @@ Tensor& sgn_out(const Tensor& self, Tensor& result) {
 Tensor sgn(const Tensor& self) { return unary_op_impl(self, at::sgn_out); }
 Tensor& sgn_(Tensor& self) { return unary_op_impl_(self, at::sgn_out); }
 
-Tensor& sin_out(const Tensor& self, Tensor& result) { return unary_op_impl_float_out(result, self, sin_stub); }
-Tensor sin(const Tensor& self) { return unary_op_impl_float(self, sin_stub); }
-Tensor& sin_(Tensor& self) { return unary_op_impl_(self, at::sin_out); }
+TORCH_IMPL_FUNC(sin_out) (const Tensor& self, const Tensor& result) {
+  sin_stub(device_type(), *this);
```

Comment:
https://github.com/pytorch/pytorch/blob/df837d0384398f1fac1000189d62b05e7ff0240d/aten/src/ATen/native/BinaryOps.h#L28

I think you'll have to update the stub structure similar to above.

Reply:
That was it, thanks!